react/features/filmstrip/functions.web.js

```diff
@@ -744,6 +745,10 @@ export function isStageFilmstripAvailable(state, minParticipantCount = 0) {
 export function isStageFilmstripEnabled(state) {
     const { filmstrip } = state['features/base/config'];
 
+    if (isVpaasMeeting(state)) {
```

Comment:
This is very odd, why are we not using a config for this?

Reply:
I don't know how to set a config only for JaaS